### ISSUE TITLE
Earth.gov - Enhance Carousel implemenation

### ISSUE THREAD

**Issue by vgeorge** [NASA-IMPACT/veda-ui#1706]
## Context

There are some points to review in the Carousel implementation on (next) Earth.gov:

* It uses a `<ul>` to wrap slides but doesn’t enforce `<li>` children, causing accessibility issues
* Next/Prior buttons share the same style and aren’t disabled when there’s no slide to move to
* Cursor should update to pointer on Next/Prior buttons
* Next/Prior buttons should have a clickable hover style, like other clickables
* Keyboard navigation doesn’t work properly
* Videos should not be interacted with when in the periphery, only selected video should be possible to play

I think we should switch to using `<div>` elements instead of `<ul>/<li>` to simplify the structure, ensuring accessibility through proper ARIA roles.

We also explored some usability issues on the Themes pages created by the really long captions. We should assess the issue and propose a design solution for Alex Gurvich.

## Acceptance Criteria

* [ ] Use `<div>` for container and slides with correct ARIA roles, dropping `<ul>/<li>`
* [ ] Next/Last buttons visibly disabled and inactive at start/end of slides
* [ ] Next/Prior buttons should have a clickable appearance (hover effect and cursor: pointer)
* [ ] Keyboard navigation works properly to control slides
* [ ] Accessibility tests confirm compliance
* [x] Proposed solution for long captions on themes and implemented
* [ ] Video is not clickable when not selected

## Definition of Done

* [ ] Unit tests
* [ ] (Where applicable) Automated Playwright tests to cover primary use case(s)
* [ ] Merged to develop in relevant repo
* [ ] (?) Tested in instance context (GHG, earth.gov, template instance, etc)
* [ ] Relevant details of work are shared in a team forum (PR, ticket, Slack, etc) to support knowledge sharing

**Comment by faustoperez** [comment]
Hi, here's a quick fix for the video carousel description issue: differences in height for the description.

- [ ] Make the arrow navigation the top element below the video so its position doesn't change
- [ ] Make the description fixed height, 5-6 text lines tall
- [ ] Add `overflow: scroll;` to the description block

<img width="1440" alt="Image" src="https://github.com/user-attachments/assets/3b4bd493-57d0-431b-8c77-a6e9a7456fff" />

**Comment by vgeorge** [comment]
Status update on this ticket:

We fixed carousel descriptions in [https://github.com/NASA-IMPACT/next-earth-gov/pull/68](https://github.com/NASA-IMPACT/next-earth-gov/pull/68) and are working on keyboard navigation in [https://github.com/NASA-IMPACT/next-earth-gov/pull/72](https://github.com/NASA-IMPACT/next-earth-gov/pull/72).

I believe we need to revisit the scope of this ticket, because refactoring the Carousel to follow WAI guidelines (as suggested by Alice [here](https://github.com/NASA-IMPACT/next-earth-gov/pull/68#issuecomment-3004080756)) will take considerable effort, as it’ll likely require a big overhaul. I suggest we ticket that separately, along with the request that “videos should not be interacted with when in the periphery...” which also seems like a heavier lift and might not be a blocker for the launch.

cc @aboydnw @AliceR @dzole0311
